src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.values(robot);
+  const keys = Object.keys(robot);
+  const res = {};
+
+  for (let i = 0; i < values.length; i++) {
+    if (values[i] in res) {
+      return null;
+    }
```

Comment:
your solution passes the checklist, but it's better to use native object methods
Checklist:
2.[CODE STYLE] - use correct check for property presence in object. Some properties can be present, but still contain falsy value.
GOOD EXAMPLE:

if (key in robot) {
EVEN BETTER EXAMPLE:

if (robot.hasOwnProperty(key)) {